### PR TITLE
Show help output on when enroll is missing required args. Fix configT…

### PR DESCRIPTION
**Pull request by plorenz**
…ypes=all processing on list services. Fixes GH-155

### CODE REVIEW

File: common/enrollment/enroll.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ const outFlag = "out"
 func NewEnrollCommand() *cobra.Command {
```
Comment (plorenz) [inline common/enrollment/enroll.go:45]: Should we just make the JWT the first positional argument to be consistent with other CLIs?

Comment (andrewpmartinez) [reply common/enrollment/enroll.go:45]: I'm leaning toward yes. If it is required for all uses of the command, it should be a required positional.

**Review by andrewpmartinez — APPROVED** [review]

File: common/enrollment/enroll.go (hunk 1 above)
Comment (plorenz) [reply common/enrollment/enroll.go]: Change made. Also marked ziti-enroller as deprecated, so we can remove it in a few releases.

**Comment by emoscardini** [comment]
@plorenz @andrewpmartinez Making the JWT positional does make sense, but we've already created wrapper scripts to enroll the edge-routers.  This change will break existing scripts already released into the production environment.   Cloud we do both?  positional or have a switch(--jwt/-j)?

**Pull request merged**